be/src/types/timestamp_value.h

```diff
@@ -109,6 +109,8 @@ class TimestampValue {
 
     bool is_valid() const;
 
+    bool is_valid_non_strict() const;
```

Comment:
comment the difference between from `is_valid` and `is_valid_non_strict`.

Reply:
for DateValue "0000-00-00", is_valid return false, is_valid_non_strict return true;